README.md

```diff
@@ -1,16 +1,88 @@
-# Title
+# Building Cells: Getter and Setter Methods
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Builds a setter method to calculate computed properties
 
 ## Introduction
 
-## SWBAT 1
+We've seen so far that we can write methods inside our classes
+that allow us to access and change instance variables:
 
-## SWBAT 2
+```js
+speak() {
+return `${this.name} says 'woof!'`
+}
+
+haveBirthday() {
+this.age += 1
+}
+```
+
+These types of methods work just fine in most cases. There is,
+however, additional JS syntax that we can use: `get` and `set`. `get` is
+used to identify methods that return calculated or dynamic data based an
+object's attributes. `set` is used for methods where we want to change an
+attribute in a controlled way. In this lab, we will be building on the
+concept of _instance variables_ to illustrate these methods.
+
```

Comment:
Don't worry about adding too much detail about this section - I'm writing a readme that will go before this and introduce get and set so this lab can assume the student has some understanding of get and set (though if you want to include reminders, that is good too)